examples/select_items/main.go

```diff
@@ -20,11 +20,11 @@ func main() {
 	ctx := context.Background()
 	client := creds.Client(ctx)
 
-	result, err := client.Clarify().SelectItems(
-		params.Query().Where(params.Comparisons{
-			"annotations.clarify/clarify-go/example/name": params.Equal("publish_signals"),
-		}).Limit(10),
-	).Do(ctx)
+	q := params.Query().
```

Comment:
We can rename `q` to `query` now as there is no `query` package causing a naming conflict.